htdocs/user/group/card.php

```diff
@@ -218,7 +218,7 @@
 			if (!empty($conf->multicompany->enabled) && !empty($conf->global->MULTICOMPANY_TRANSVERSE_MODE)) {
 				$object->entity = 0;
 			} else {
-				$object->entity = GETPOST("entity");
+				$object->entity = GETPOSTISSET("entity") ? GETPOST("entity") : $conf->entity;
```

Comment:
If we don't want to modify it, we should have
$object->entity = GETPOSTISSET("entity") ? GETPOST("entity") : $object->entity;
instead of
$object->entity = GETPOSTISSET("entity") ? GETPOST("entity") : $conf->entity;
??